spec/cases/store_attribute_spec.rb

```diff
@@ -135,11 +120,13 @@
       expect(jamie).to be_active
       expect(jamie.salary).to eq 12
 
+      jamie.salary = 13
```

Comment:
Without updating store contents, Active Record doesn't include this attribute into a changeset (thus, it's not updated).